### DISCUSSION TITLE
Can lonboard use deck.gl's GlobeView?

### DISCUSSION THREAD

**Discussion by tylere** [developmentseed/lonboard#570] [Q&A] [answered]
Even in its current "experimental" state, deck.gl's [GlobeView](https://deck.gl/docs/api-reference/core/globe-view) class is useful for displaying datasets with continental to global coverage. Can lonboard use this view class?

**Comment by kylebarron** [comment, marked as answer]
No, not today, but that's a potential addition for the future. It's a little tricky because today we always render a maplibre map in conjunction with deck.gl, but maplibre doesn't work in globe view. So it requires either choosing to not use maplibre or allowing some rendering without maplibre (https://github.com/developmentseed/lonboard/pull/437)
